drivers/filesystems/ext2/src/ext4/ext4_extents.c

```diff
@@ -14,9 +14,13 @@
  */
 
 #include "ext2fs.h"
+#ifdef __REACTOS__
 #include <linux/ext4.h>
```

Comment:
```suggestion
#include "linux/ext4.h"
```

Reply:
I didn't wanted to change anything in our code, except the ifdef.
but I suppose I can do that change